docs/contributing.md

```diff
@@ -33,4 +33,68 @@ git remote add jaril git@github.com:jaril/webreplay-devtools.git
 
 ```bash
 git push jaril readme-changes
-```
\ No newline at end of file
+```
+
+## Issue Labeling Guide
+
+### **Parent Categories**
+
+Every issue will have at least a parent category and may have a more granular child category.
+
+The more granular categories would be labeled according to this list. Underlined is the name of the label and below it is the list of what that label encompasses.
+
+### **Overarching category labels:**
+*Color code for these labels are #D876E3*
+
+<u>Bug</u>
+- All bugs
+
+<u>Feature</u>
+- New feature requests
+
+<u>Tooling</u>
+- Tests
+- Logrocket
+
+<u>UI Polish</u>
+- Anything UI that isn't broken or functionality related
+
+<u>Performance</u>
+- Lagging and other performance issues
+
+
+### **Labels for Bug or Feature:**
+*Color code for these labels are #CCEBFF*
+
+<u>Debugger</u>
+- Sidebar Panel and all its contents
+- Accordions
+- Editor
+
+<u>Viewer</u>
+- Issues related *specifically* to the viewer that don't carry over to Debug mode
+
+<u>Comments</u>
+- Comments / Transcripts in both View and Debug mode
+
+<u>Elements</u>
+- Anything within the Elements panel
+- Inspector
+
+<u>Console</u>
+- Anything within the Console Panel
+
+<u>Timeline</u>
+- The Timeline in both View and Debug mode
+
+<u>Video</u>
+- The Video player in both View and Debug mode
+
+<u>Account</u>
+- User's dashboard and profile
```

Comment:
It'd be nice to avoid duplication with the labels descriptions.

I'd suggest replacing this content with a short description of how we think about lables and then link to the labels page.

I love having the examples below.

Reply:
just pushed a refactor, i removed the individual labels and added a link to them along with a description. also added a few more examples.